### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by isopov**
Depends on #20
I suggest run continious integration on https://travis-ci.com since their plan is to unify their .com and .org resources and leave .com only - https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/
Build in my fork - https://travis-ci.com/isopov/checkinx-utils/builds/114542240
I have added a badge in README that points to not yet existing https://travis-ci.com/dsemyriazhko/checkinx-utils

### CODE REVIEW

**Comment by isopov** [comment]
BTW in the first build on openjdk8 this issue is seen - https://travis-ci.community/t/using-jdk-openjdk8-brings-javac-9-0-1 and I believe it is already reported and known to travis team

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: java
```
Comment (dsemyriazhko) [inline .travis.yml]: I haven't used travis before, but the time has come. Are you sure of java?) project is written in kotlin.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+jdk: 
+  - openjdk8
```
Comment (dsemyriazhko) [inline .travis.yml]: I suppose, only 8 supported great by kotlin

Comment (isopov) [reply .travis.yml]: Actually java6-12 are supported by Kotlin (Just that kolin does not use any features of java9+ and does nt compile to those bytecode levels, yet) so I think it is worth testing this support for this library on CI.

File: .travis.yml (hunk 1 above)
Comment (isopov) [reply .travis.yml]: >Are you sure of java?) project is written in kotlin.

But for the JVM ;-) I suppose you do not plan to make this project backend-agnostic like kotlinx.serialization, but if you do - travis config is not set in stone

**Comment by isopov** [comment]
Chaned badge to point to https://travis-ci.com/TinkoffCreditSystems/checkinx-utils

**Pull request merged**